src/main/java/io/cbitler/stealingartefacts/StealingArtefactsOverlay.java

```diff
@@ -54,6 +55,25 @@ public Dimension render(Graphics2D graphics) {
                 panelComponent.getChildren().add(LineComponent.builder().left(String.valueOf(plugin.artefactsToGoal)).build());
             }
 
+            panelComponent.getChildren().add(LineComponent.builder().build());
+            panelComponent.getChildren().add(LineComponent.builder().left("Guard Lures:").build());
```

Comment:
In the spirit of making stuff configurable, do you think there should be a config option for this?